### DISCUSSION TITLE
can/could super-linter automatically fix issues?

### DISCUSSION THREAD

**Discussion by mr-blanket** [super-linter/super-linter#5866] [Ideas]
Hi super-linter team!

I know not all tools are capable of automatically fixing the code issues. But some tools e.g. `isort`, `black` are.

Would it be possible to add a switch e.g. `AUTO_FIX=true` which would instead of checking for compliance, automatically fix code issues using tools capable of that (and are enabled)?

What are your thoughts?

Thanks!

**Comment by ferrarimarco** [comment]
I can see how a "fix mode" can be useful. We'll happily review PRs to implement this.
